DependencyInjection/Configuration.php

```diff
@@ -51,6 +51,12 @@ public function getConfigTreeBuilder()
                                     ->append($this->createConsumersNodeDefinition())
                                 ->end()
                             ->end()
+                            ->arrayNode('old_rabbitmq')
```

Comment:
I prefer `rabbitmq` you you are agree that you should hide the implementation (composer dependency) by a clear interface upfront, this way it's perfectly swappable, you'll be able to replace this dependency by another one at any time.